wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -863,6 +888,7 @@ public void shutdownAndDeleteWallet() {
 
     public void finalizeWipe() {
         cancelScheduledStartBlockchainService();
+        WorkManager.getInstance(this.getApplicationContext()).cancelAllWork();
```

Comment:
There was a bug when after the wipe the CrowdNode signup worker would still persist.
In the case of the Explore worker, we should cancel it too. Given that during the wipe all of the Explore settings are cleared, it can lead to undefined behavior of the sync job.